api/src/main/resources/workbench-api.yaml

```diff
@@ -35,20 +35,35 @@ host: api.pmi-ops.org
 securityDefinitions:
   aou_oauth:
     x-aou-note: |
-      Establish the fact that *some endpoints* are OAuth protected
-      by defining an `aou_oauth` security mode, which we'll assigning
-      to any protected endpoints below.
+      Users authenticate with our API using a Google-issued OIDC access token.
 
-      TODO: Vet/fix this token and/or authorization URL to work in practice.
-      These are currently included simply to satisfy the Swagger specification,
-      as this is not directly used to dictate oauth details (just used to
-      annotate which methods require oauth).
+      This is a non-standard use of Swagger 2.0's oauth2 type. The aou_auth
+      entry under security (below) serves to establish the fact
+      that all endpoints are protected. This annotation ensures that client
+      libraries know to send bearer tokens when calling, but otherwise only
+      provides a minimal OAuth2 implementation. Future Swagger versions may
+      provide a better mapping to our authorization and authentication flows.
 
-      The aou_auth entry under security (below) serves to establish the fact
-      that all endpoints are protected: this annotation ensures that client
-      libraries know to send bearer tokens when calling
+      We use the OAuth2 authorization code grant with the the PKCE extension on
+      public clients. However, Google only supports the authorization code grant
+      for confidential clients. To enable public client access, we provide an
```

Comment:
It's [OAuth-specific terminology](https://oauth.net/2/client-types/). I think that's less obvious than terms like "PKCE," though.

I'm not sure what level of detail to go into here, since I'll be more thorough in the docs I write. Maybe I should reference the docs in this comment for anyone seeking additional info?

Reply:
Added a couple links 👍